api/src/pcapi/routes/serialization/users.py

```diff
@@ -210,9 +210,8 @@ class ProFlagsQueryModel(BaseModel):
 
 
 class SubmitReviewRequestModel(BaseModel):
-    isPleasant: bool
-    isConvenient: bool
-    comment: str
+    userSatisfaction: str
```

Comment:
```suggestion
    userSatisfaction: Literal["Très mauvaise", "Mauvaise", "Correcte", "Bonne", "Excellente"]
```

Pour ne pas permettre à nos utilisateurs de stocker n’importe quoi dans nos logs